src/rules/no-single-line-comment-block.ts

```diff
@@ -18,12 +17,13 @@ const hasSpecialCases = (
   ignoreRegex: RegExp[],
 ): boolean => {
   const specialCases = [
+    'c8',
+    'cspell',
```

Comment:
This is really not needed, the special cases here are meant to be really generic and only have Eslint directives, without any assumption as to what any consumer might use.

Actually `c8` should also not be here.

The way to handle this should be at the consumer level in the Eslint configuration for this plugin, see here:

https://github.com/ridedott/eslint-config/blob/c94f78966d51b2be7270d494ed67c07aaa968ad4/rules/ridedott.js#L2-L8

Reply:
That means that overrides will be needed in each and every repository. If there are common tools that are used frequently, it would be a good idea to have them here for seamless integration, wouldn't it? Should make everyone's lives easier 🤷‍♂️

Reply:
> That means that overrides will be needed in each and every repository.

we do use [eslint-config](https://github.com/ridedott/eslint-config) as a central configuration right?
This means it only changes there, and it is already there :D

Reply:
Fair enough 🙂